README.md

```diff
@@ -225,6 +225,24 @@ Note that you can also create an automatically archived session, by passing in `
 as the `archiveMode` option when you call the `OpenTok.createSession()` method (see "Creating
 Sessions," above).
 
+For composed archives, you can set change the layout dynamically, using the `OpenTok.setArchiveLayout(archiveId, type, stylesheet, callback)` method:
+
+```javascript
+opentok.setArchiveLayout(archiveId, type, null, function (err) {
+  if (err) return res.send(500, 'Could not set layout ' + type + '. error=' + err.message);
```

Comment:
For the error, I think we should just follow the convention we've been following:
` if (err) console.log(err)` instead of sending a status with an error

Reply:
The property in the REST method is called `type`. https://tokbox.com/developer/rest/#change_composed_archive_layout

Also, our docs refer to layout *types*: https://tokbox.com/developer/guides/archiving/layout-control.html.